src/main/features/testing-support/routes/create-draft.ts

```diff
@@ -0,0 +1,41 @@
+import * as express from 'express'
+
+import { Paths } from 'testing-support/paths'
+import { Paths as DraftSubmittion } from 'claim/paths'
+import { ErrorHandling } from 'common/errorHandling'
+
+import { DraftService } from 'services/draftService'
+import { User } from 'idam/user'
+import { DraftClaim } from 'drafts/models/draftClaim'
+import { claimDraft as claimDraftData } from '../../../../test/data/draft/createDraft'
+import { Draft } from '@hmcts/draft-store-client'
+import moment = require('moment')
+
+const draftService = new DraftService()
+
+function getDraftType (req: express.Request): string {
+  return Object.keys(req.body.action)[0]
+}
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.createDraftPage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
+      res.render(Paths.createDraftPage.associatedView)
+    })
+  )
+  .post(Paths.createDraftPage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
+      const user: User = res.locals.user
+      const drafts = await draftService.find(getDraftType(req), '100', user.bearerToken, (value) => value)
+
+      drafts.forEach(async draft => {
+        await new DraftService().delete(draft.id, user.bearerToken)
```

Comment:
This ts file uses much of the same code as `delete-drafts` to potentially delete any pre-existing claim drafts, to be replaced by the generic template.
If there is a better way to implement this more generically, please comment